setup.py

```diff
@@ -17,7 +17,8 @@
     packages=setuptools.find_packages(),
     install_requires=[
         'omero-py',
-        'numpy'
+        'numpy',
+        'dataclasses'
     ],
-    python_requires='>=3.6'
+    python_requires='<3.7'
```

Comment:
This needs to be combined with the dataclasses line (or alternatiely, done as an `extra_require`).

Reply:
Then I just leave this change as it was? I wouldn't know. In the examples they use 3.6 indeed.